src/server/server_family.cc

```diff
@@ -448,6 +449,15 @@ void ServerFamily::Shutdown() {
     snapshot_fiber_.Join();
```

Comment:
now I see that `snapshot_fiber_ ` and `is_snapshot_done_` are confusing names.
Can you please rename them to `snapshot_schedule_fb_` and `schedule_done_` accordingly?